app/views/api/alerts/index.html.slim

```diff
@@ -0,0 +1,59 @@
+- content_for :page_header_title, presenter.page_title
+
+- if presenter.empty_state?
+  tr role="row"
+    td role="cell" colspan="100"
+      div class="pf-c-empty-state"
+        div class="pf-c-empty-state__content"
+          i class="fas fa-check-circle pf-c-empty-state__icon" aria-hidden="true"
+          h1 class="pf-c-title pf-m-lg"
+            = t('.empty_state.title')
+          div class="pf-c-empty-state__body"
+            = t('.empty_state.body')
+
+- else
+  - content_for :javascripts do
+    = javascript_packs_with_chunks_tag 'table_toolbar'
+
+  table class="pf-c-table pf-m-grid-lg" role="grid" aria-label="Alerts table" data-toolbar-props=presenter.toolbar_props.to_json
+    thead
+      tr role="row"
+        th role="columnheader" scope="col" Account
+        th role="columnheader" scope="col" Application
+        th role="columnheader" scope="col" Message
+        = th_sortable 'level', 'Level'
+        = th_sortable 'timestamp', 'Time (UTC)'
+        td
+
+    tbody role="rowgroup"
+      - if presenter.empty_search?
+        = render partial: 'shared/empty_search_state', locals: { scope: 'api.alerts.index' }
+      - else
+        - confirm_delete = t('.delete_alert_confirmation')
+        - presenter.alerts.each do |alert|
+          - cinstance = alert.cinstance
+          tr id=dom_id(alert) role="row" class=alert.state
+            td role="cell" data-label="Account"
+              = presenter.link_to_account_for_alert(alert)
+            td role="cell" data-label="Application"
+              = link_to cinstance.name, provider_admin_application_path(cinstance)
+            td role="cell" data-label="Message"
+              = h alert.message
+            td role="cell" data-label="Level" class="utilization"
+              = colorize_utilization(alert.level)
+            td role="cell" data-label="Time (UTC)"
+              = l alert.timestamp
+            td role="cell" class="pf-c-table__action"
+              div class="pf-c-overflow-menu"
+                div class="pf-c-overflow-menu__content"
+                  div class="pf-c-overflow-menu__group pf-m-button-group"
+                    div class="pf-c-overflow-menu__item"
+                      = delete_button_for polymorphic_path([:admin, @service, alert]), method: :delete,
```

Comment:
Here and in `action_link_to` would it be better to use `presenter.service` ?
Doesn't make any difference in this case, but just to keep it more consistent...

Reply:
IDK, maybe. A presenter encapsulates logic used to render the data in the view, service however is just an instance variable.